class/hd-segwit-bech32-wallet.js

```diff
@@ -53,44 +54,47 @@ export class HDSegwitBech32Wallet extends AbstractHDWallet {
   }
 
   async generate() {
-    const that = this;
-    return new Promise(function(resolve) {
+    return new Promise(resolve => {
       if (typeof RNRandomBytes === 'undefined') {
         // CLI/CI environment
         // crypto should be provided globally by test launcher
-        return crypto.randomBytes(32, (err, buf) => {
+        return crypto.randomBytes(HDSegwitBech32Wallet.randomBytesSize, async (err, buf) => {
           // eslint-disable-line
```

Comment:
what is eslint problem here?

Reply:
there is no error, someone had to add it unnecessary, I'll remove it